src/controllers/trainee/controller.ts

```diff
@@ -35,14 +43,14 @@ class TraineeController {
         try {
             console.log('Inside put function of trainee Controller');
            let resp = await this.userRepository.update(req.body.dataToUpdate);
-             if(resp) {
+             if (resp) {
                 console.log('Response of Repo is', resp);
                 res.send({
+                    status:"ok",
                     message: 'trainee updated sucessfully',
                     data: resp
                 });
             }
-           
         } catch (err) {
             console.log('Inside err', err);
```

Comment:
Send a proper response when any unexpected error occurred, You can use next 
() like next({code:500,message: "message" ,error: "error mesasage"}